scripts/github-last-contribution.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import argparse
+import urllib.request
+from bs4 import BeautifulSoup
+'''
+For using this script call the get_usernames() function and pass all the
+usernames in a space separated string.
+'''
+
+
+def last_contribution(user_name):
+    profile_url = "https://github.com/{}/zulip".format(user_name)
+    try:
+        profile_page = BeautifulSoup(
+            urllib.request.urlopen(profile_url), "lxml")
+    #404 page not reachable or page doesn't exist.
+    except urllib.error.URLError as e:
+        raise e
+    time_tag = profile_page.findAll('relative-time', datetime=True)
+    # I am retrieving the datetime data below and splitting it into date and
+    # time.
+    time_stamp = time_tag[0].get("datetime")[:-1].split("T")
+    name_and_time_stamp = [user_name] + time_stamp
+    return name_and_time_stamp
+
+
+def get_usernames(user_names):
+    user_names = user_names.split(' ')
+    print("Here are the timestamps for the last commit to the Zulip forks:")
+    for user_name in user_names:
+        try:
+            name_and_time_stamp = last_contribution(user_name)
+            print("{} last updated on {}, {}".format(*name_and_time_stamp))
+        except:
```

Comment:
Here you should be more specific (you want to catch just particular kind of errors)

`except MyError:`